src/scale_audio_rating.py

```diff
@@ -292,7 +312,8 @@ async def main(cfg, args):
                     ],
                 },
             ],
-            "metadata": metadata
+            "metadata": metadata,
+            "tags":[file_urls, args.project]

```

Comment:
Is `file_urls` an array? Either way, that should probably be the `unique_id` given it would change per task. 

The `args.project` is repetitive as a tag. We should look at what's in the metadata and see if there's some other info we could pull out for the tags.